python/s3-exposed-bucket.py

```diff
@@ -0,0 +1,84 @@
+#
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+#
+# Ensure that no buckets are globally accessible.
+# Description: Checks that all buckets restrict access from the global groups.
+#
+# Trigger Type: Change Triggered
+# Scope of Changes: S3:Bucket
+#
+# NOTE: This rule will only find applicable resources once S3 resources are supported by AWS Config.
```

Comment:
Thank you very much for your comment. You are right, I agree that it could be confusing for some users, I have updated the PR without this rule. I'll save it for when S3 buckets become officially supported by AWS Config.